tests/snapd

```diff
@@ -59,20 +58,12 @@ cleanup() {
 }
 trap cleanup EXIT HUP INT TERM
 
-# Configure to use the proxy
-curl -s http://canonical-lxd.stgraber.org/config/snapd.sh | sh
-
-if [ "${track}" = "latest" ]; then
-    snapd_channel=${channel}
-else
-    snapd_channel=${track}/${channel}
-fi
-
-echo "==> Installing LXD snap from ${track}/${channel}"
+echo "==> Installing LXD snap from ${LXD_SNAP_CHANNEL}"
 snap remove lxd || true
 snap version
 snap list
-snap install lxd --channel="${snapd_channel}" --cohort=+
+snap install lxd --channel="${LXD_SNAP_CHANNEL}" --cohort=+
```

Comment:
There is the `install_lxd` helper that should replace most of those.

Reply:
Did see this helper, but in this case we need to preseed before starting the daemon, so we can't use `install_lxd` because it runs `waitready` after installation.

Reply:
Could you update `install_lxd()` to accept an argument on whether it should run `waitready` or not?
Also, adding `--cohort=+` to the `install_lxd()` would make a lot of sense.

I'm coming back to this due to the `snap remove lxd || true` lacking the `--purge` :)

Reply:
Updated